### ISSUE TITLE
digits missing on typescript types.

### ISSUE THREAD

**Issue by eaguilera23** [freeall/currency-codes#17]
# Expected behaviour

```typescript
import * as currencyCode from 'currency-codes';

currencyCode.code('EUR').digits
// 2
```

# Actual behaviour

```typescript
import * as currencyCode from 'currency-codes';

currencyCode.code('EUR').digits

// Property 'digits' does not exist on type 'CurrencyCodeRecord'
```

# Solution

It should be added on `./index.d.ts` as

```typescript
declare module "currency-codes" {
  export interface CurrencyCodeRecord {
    code: string;
    number: number;
    currency: string;
    countries: string[];
    // HERE: digits: number;
  }

// ...
```

# Disclosure

I can add it myself, but javascript/typescript is not my main language, so basically I don't know what I am doing. I would like confirmation from someone with more experience.

Thank you.

**Comment by kloener** [comment]
Looks good :)

**Comment by freeall** [comment]
Looks good, thanks!

**Comment by freeall** [comment]
Released as `1.5.0`
